lib/posix-process/Config.uk

```diff
@@ -26,6 +26,23 @@ config LIBPOSIX_PROCESS_SIGNAL
 	select LIBSYSCALL_SHIM
 	select LIBUKLOCK
 
+menuconfig LIBPOSIX_PROCESS_BRK
+	bool "brk() system call and sbrk() libc function"
+	select LIBPOSIX_PROCESS_PIDS
```

Comment:
These need to be updated because of the changes we made in `0.20.0`: 
* `CONFIG_LIBPOSIX_PROCESS_PIDS`: Replace with `CONFIG_LIBPOSIX_PROCESS_MULTITHREADING`. 
* `CONFIG_LIBPOSIX_PROCESS_INIT_PIDS`: Remove.